web_ui/src/pages/annotator/components/sidebar/dataset/dataset-picker/dataset-picker.test.tsx

```diff
@@ -49,12 +60,11 @@ describe('DatasetPicker', () => {
 
         expect(screen.queryByRole('option', { hidden: true, name: 'Active set', selected: true })).toBeInTheDocument();
 
-        fireEvent.click(screen.getByRole('button', { name: /Choose annotation dataset/ }));
+        const picker = screen.getByRole('button', { name: /Choose annotation dataset/ });
+        const selectTester = testUtilUser.createTester('Select', { root: picker });
```

Comment:
According to the documentation, this is the recommended way to interact with the picker https://react-spectrum.adobe.com/react-spectrum/Picker.html#testing